spring-security-modules/spring-security-web-rest-custom/src/main/java/com/baeldung/config/MainWebAppInitializer.java

```diff
@@ -2,33 +2,26 @@
 
 import java.util.Set;
 
-import javax.servlet.FilterRegistration.Dynamic;
-import javax.servlet.ServletContext;
-import javax.servlet.ServletException;
-import javax.servlet.ServletRegistration;
-
 import org.springframework.web.WebApplicationInitializer;
 import org.springframework.web.context.ContextLoaderListener;
 import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
 import org.springframework.web.filter.DelegatingFilterProxy;
 import org.springframework.web.servlet.DispatcherServlet;
 
+import jakarta.servlet.FilterRegistration.Dynamic;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.ServletRegistration;
+
 public class MainWebAppInitializer implements WebApplicationInitializer {
 
     public MainWebAppInitializer() {
         super();
     }
 
-    //
-
-    /**
-     * Register and configure all Servlet container components necessary to power the web application.
-     */
     @Override
-    public void onStartup(final ServletContext sc) throws ServletException {
+    public void onStartup(final ServletContext sc) {
         System.out.println("MyWebAppInitializer.onStartup()");
 
-        // Create the 'root' Spring application context
         final AnnotationConfigWebApplicationContext root = new AnnotationConfigWebApplicationContext();
         root.scan("com.baeldung.config.parent");
         // root.getEnvironment().setDefaultProfiles("embedded");
```

Comment:
Do we know why we have this commented code here? Maybe we can remove this.

Reply:
Like maybe it is mentioned in the article or so.

Reply:
In fact, there is no mention. Therefore, removing.